src/sample_weight_audit/test_on_representative_sample.py

```diff
@@ -2,12 +2,12 @@
 
 import numpy as np
 import matplotlib.pyplot as plt
-from scipy.stats import kstest, mannwhitneyu, ttest_ind
 from scipy.sparse import csr_array
 
 
 from .generate_weighted_and_repeated_data import get_estimator_dataset
 from .multifit import multifit_over_weighted_and_repeated
+from .pval_tests import scan_for_pvalue
 
 
 def paired_test(
```

Comment:
"Paired" test has a specific meaning in statistics, and this is not what we do here. Let's call that "weighted_repeated_fit_equivalence_test" instead.